routes/users.js

```diff
@@ -111,4 +111,6 @@ router.post(
   usersHandler.blockAnonUserFromChat
 );
 
+router.get('/check-allow-dm/:post_id', auth.isAuth, usersHandler.checkAllowAnonDM);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new route `/check-allow-dm/:post_id` seems to be checking if anonymous direct messages are allowed for a user based on a post ID. However, the route parameter is named `:post_id`, which might be misleading as it suggests that the route is related to posts, not users. If this route is intended to check a user's settings, consider renaming the parameter to `:user_id` for clarity.
```diff
- router.get('/check-allow-dm/:post_id', auth.isAuth, usersHandler.checkAllowAnonDM);
+ router.get('/check-allow-dm/:user_id', auth.isAuth, usersHandler.checkAllowAnonDM);
```
Please ensure that the corresponding changes are made in the handler function `checkAllowAnonDM` to accommodate this change.


<!-- This is an auto-generated comment by OSS CodeRabbit -->